src/chrome/content/rules/Tryton.org.xml

```diff
@@ -0,0 +1,41 @@
+<!--
+	Nonfunctional hosts in *.tryton.org:
+		- demo*.tryton.org (m) (subdomain for each version)
+		- doc.tryton.org (m)
+		- tub2013.tryton.org (m)
+		- tub2016.tryton.org (m)
+		- tuba2015.tryton.org (m)
+		- tul2014.tryton.org (m)
+		- videos.tryton.org (m)
+		- debian.tryton.org (m)
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
```

Comment:
Also, since all problematic subdomains are mismatched, probably it would make more sense to write it in this way:
```xml
<!--
	Mismatched:
		- subdomain1
		- subdomain2
		- subdomain3
-->
```